src/debug-adapter/client.ts

```diff
@@ -229,6 +465,38 @@ class BazelDebugSession extends DebugSession {
     return bazelExecutable;
   }
 
+  /**
+   * Invokes {@code bazel info} and returns the information in a map.
+   *
+   * @param bazelExecutable The name/path of the Bazel executable.
+   * @param cwd The working directory in which Bazel should be launched.
+   */
+  private getBazelInfo(bazelExecutable: string, cwd: string): Promise<Map<string, string>> {
+    return new Promise((resolve, reject) => {
+      const execOptions = {
+        cwd: cwd,
+        maxBuffer: 500 * 1024
```

Comment:
This means "bazel info" may not return more than 500kB? Add a comment about it

Reply:
Done.